sdk/appplatform/Azure.ResourceManager.AppPlatform/samples/Generated/Samples/Sample_AppPlatformAppResource.cs

```diff
@@ -249,7 +249,7 @@ public async Task Update_AppsUpdateVNetInjection()
         public async Task GetResourceUploadUri_AppsGetResourceUploadUrl()
         {
             // Generated from example definition: specification/appplatform/resource-manager/Microsoft.AppPlatform/stable/2022-12-01/examples/Apps_GetResourceUploadUrl.json
-            // this example is just showing the usage of "Apps_GetResourceUploadUri" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "Apps_GetResourceUploadUrl" operation, for the dependent resources, they will have to be created separately.
```

Comment:
`Url` is the original ID: https://github.com/Azure/azure-rest-api-specs/blob/c852a76b96e5c71972cdc62a303665fec7c867b6/specification/appplatform/resource-manager/Microsoft.AppPlatform/stable/2023-12-01/appplatform.json#L4213